meerk40t/core/spoolers.py

```diff
@@ -552,15 +617,20 @@ def command(self, *job, priority=0, helper=True):
             self._lock.notify()
         self.context.signal("spooler;queue", len(self._queue))
 
-    def send(self, job):
+    def send(self, job, prevent_duplicate=False):
```

Comment:
We need to be able to put the same job into the spooler repeatedly since we'll stream as such.